### ISSUE TITLE
Tags properties could use a Map

### ISSUE THREAD

**Issue by senzzzi** [statful/statful-client-springboot#3]
The tags that can be added by properties could use a map, so this:

>statful.client.springboot.metrics.tags[0].name=application
>statful.client.springboot.metrics.tags[0].value=starter
>statful.client.springboot.metrics.tags[1].name=framework
>statful.client.springboot.metrics.tags[1].value=springboot

could actually be written like:

>statful.client.springboot.metrics.tags[application]=starter
>statful.client.springboot.metrics.tags[framework]=springboot

**Comment by miguelcnf** [comment]
Yes I agree it might be clearer. Not sure if we'll have time to do it any time soon. 

It might be a matter of refactoring the `SpringbootClientConfiguration.Metrics` class and it's getter usages tho.
